charmhelpers/contrib/openstack/context.py

```diff
@@ -768,6 +768,8 @@ def __call__(self):
         if send_notifications_to_logs:
             ctxt['send_notifications_to_logs'] = send_notifications_to_logs
 
+        ctxt['rabbit_heartbeat_timeout_threshold'] = conf.get('rabbit-heartbeat-timeout-threshold')
```

Comment:
All of the above versions of conf.get() include the default 'None' parameter.  For consistency, we should probably include that, please.

Reply:
done